ducktape/mark/_mark.py

```diff
@@ -411,6 +435,27 @@ def ignorer(f):
     return ignorer
 
 
+def ok_to_fail(*args, **kwargs):
+    """
+    Test method decorator which signals to the test runner to run test but ignore result.
```

Comment:
nit: I'm not sure I'd say the result is ignored, as opposed to it not affecting the success of the overall test run.

Reply:
i'll update the wording and take this out of draft.